UefiPayloadPkg/UefiPayloadPkg.fdf

```diff
@@ -333,6 +337,31 @@ INF ShellPkg/DynamicCommand/DpDynamicCommand/DpDynamicCommand.inf
 INF ShellPkg/Application/Shell/Shell.inf
 !endif
 
+[FV.SECFV]

```

Comment:
Wrap this with `!if $(UNIVERSAL_PAYLOAD) == TRUE`
Similar to https://github.com/tianocore/edk2/blob/ce60f1755f5465a8b53c303f1f8c8348a33bcbc7/UefiPayloadPkg/UefiPayloadPkg.fdf#L62